entries/b/bitvavo.com.json

```
@@ -0,0 +1,48 @@
+{
+  "bitvavo": {
+    "domain": "bitvavo.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.bitvavo.com/hc/en-us/articles/4405125367057",
+    "categories": [
+      "cryptocurrencies"
+    ],
+    "regions": [
+      "at",
+      "be",
+      "bg",
+      "ch",
+      "cy",
+      "cz",
+      "de",
+      "dk",
+      "ee",
+      "es",
+      "fi",
+      "fr",
+      "gb",
```

```suggestion
      "gb",
      "gi",
```

They also allow residents of Gibraltar.

>  Opening or using an Account is restricted to individuals or legal entities (represented by an authorised individual) who are resident or established in the European Economic Area, Gibraltar, Switzerland or the United Kingdom.

---

Didn’t think Gibraltar had its own country code 👀